lib/internal/repl.js

```diff
@@ -59,7 +61,17 @@ function createRepl(env, cb) {
 }
 
 function setupHistory(repl, historyPath, ready) {
-  const fs = require('fs');
+  // default to using <homedir>/tmp/node_history_file
```

Comment:
*shrug* I wasn't really sure where to put it.

`<homedir>/.node_history` does sound probably more reasonable. Any idea how this would interact with windows? Hah.